qa/suites/fs/workload/copyfrom/yes.yaml

```diff
@@ -0,0 +1,3 @@
+overrides:
+  kclient:
+    mntopts: ["copyfrom"]
```

Comment:
Unfortunately, it doesn't look like these options will get added to the xfstests config file. The code in xfstests_dev.py seems to only add this:

            export TEST_FS_MOUNT_OPTS="-o name=admin,secret={}"

It'd be nice to have this append the options from the yaml files too, if that's possible. Also, note that it also doesn't bother configuring a scratch mount. Some xfstests require that, so it might also be nice to add that in.

Reply:
Right, looks like the xfstests support requires a bit more things than I had initially anticipated.  I'll see what I can come up with, but I'll need to spend some (good amount of) time re-learning how to execute manually tests in teuthology.